build.sbt

```diff
@@ -126,3 +126,46 @@ lazy val benchmarks = project
   .dependsOn(core.jvm)
   .settings(name := "lucuma-core-benchmarks")
   .enablePlugins(NoPublishPlugin, JmhPlugin)
+
+// for publishing CSS to npm
+lazy val npmPublish = taskKey[Unit]("Run npm publish")
+lazy val npm        = project
+  .in(file("modules/npm"))
+  .dependsOn(core.js)
+  .enablePlugins(ScalaJSPlugin, NoPublishPlugin)
+  .settings(
+    scalaJSLinkerConfig ~= (_.withModuleKind(ModuleKind.ESModule)),
+    npmPublish := {
+      import scala.sys.process._
+      val _      = (Compile / fullLinkJS).value
+      val outDir = (Compile / fullLinkJS / scalaJSLinkerOutputDirectory).value
+      IO.write(
+        outDir / "package.json",
+        s"""|{
+            |  "name": "lucuma-core",
```

Comment:
This will publish here:
https://www.npmjs.com/package/lucuma-core

Reply:
We need credentials right?

Reply:
Already set up.